### PR TITLE
[pt-br] Add docs/tasks/configure-pod-container/extended-resource.md

### PR DESCRIPTION
**Pull request by DonatoHorn**

### CODE REVIEW

File: content/pt-br/docs/tasks/configure-pod-container/extended-resource.md
Comment (sftim) [inline content/pt-br/docs/tasks/configure-pod-container/extended-resource.md:15]: If writing the translated term “Contêiner”, don't capitalize the first letter.

File: content/pt-br/docs/tasks/configure-pod-container/extended-resource.md
Comment (sftim) [inline content/pt-br/docs/tasks/configure-pod-container/extended-resource.md:7]: Git tracks contributions. No need to add this field.

Comment (DonatoHorn) [reply content/pt-br/docs/tasks/configure-pod-container/extended-resource.md:7]: Hi, I'm new contributor on pt-br docs, so my difficults are identify pages (original version) than where used base-line for internacionalizations, in pt-br docs we have pages from v1.18, 1,19, 1.20, 1.21, 1.22, 1.23, 1.24, how can I guess what pages need update. I'ts nice to do this, make a diff in the english docs, so the pages that need update will be shown. But in pt-br we don't have versioning, so if I have a origin_version  in the docs I can use a grep to show what a version used in, and assign the contributor to ask if interested in update the docs they translated.

Comment (sftim) [reply content/pt-br/docs/tasks/configure-pod-container/extended-resource.md:7]: I would ask folks in the Slack channel for localizations for help on doing this. We have tools that you can use, and I hope someone will be happy to show you.

Comment (stormqueen1990) [reply content/pt-br/docs/tasks/configure-pod-container/extended-resource.md:7]: This issue with outdated localization was previously discussed in issue #29649. Would having automation like this require some kind of RFC to be implemented?

File: content/pt-br/docs/tasks/configure-pod-container/extended-resource.md
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/extended-resource.md:21]: Suggestion: use italic formatting here instead of code formatting (_dongle_ instead of `dongle`).

File: content/pt-br/docs/tasks/configure-pod-container/extended-resource.md
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/extended-resource.md:37]: Same suggestion here. It makes more sense to use italic as it is a term from a different language, rather than a coding-specific term.

File: content/pt-br/docs/tasks/configure-pod-container/extended-resource.md
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/extended-resource.md:57]: Same suggestion here.

File: content/pt-br/docs/tasks/configure-pod-container/extended-resource.md
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/extended-resource.md:82]: Missing colon after "descreva o pod".

**Review by stormqueen1990 — CHANGES_REQUESTED** [review]
Hi @DonatoHorn, thanks for your contribution! 🎉 

Very minor changes required here. Mainly:
- There's a missing colon in one expression.
- I suggest that you change the formatting for the term dongle from code-formatting to italic (_dongle_ rather than `dongle`).

After these are addressed, it will be a LGTM.

Thanks again!

File: content/pt-br/docs/tasks/configure-pod-container/extended-resource.md
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/extended-resource.md:74]: Minor: two code formattings leftover here.

**Comment by stormqueen1990** [comment]
Hello there, @DonatoHorn! 👋

This page requires a few very minor adjustments before we can merge it. Do you feel comfortable addressing these changes so we can move this contribution forward?

Thanks in advance! 😃

**Comment by stormqueen1990** [comment]
/remove-lifecycle rotten
/lgtm
/cc @edsoncelio
Could you please provide a second review?

**Comment by stormqueen1990** [comment]
/retitle [pt-br] Add docs/tasks/configure-pod-container/extended-resource.md

**Review by stormqueen1990 — COMMENTED** [review]
Hi there, @DonatoHorn! 👋 

Thank you very much for addressing the previous feedback! 😃 

It's LGTM. It does look like it's not building correctly due to a change in the `main` branch. Could you please rebase this branch off of `main`, so we can check it's building correctly?

I'll leave my approval anyways as this is looking good to be closed.

/approve

**Comment by stormqueen1990** [comment]
/cc @MrErlison 
Could you please provide a second review?

**Comment by DonatoHorn** [comment]
Hi, I just do the minor formating in the lines 73 & 74 Kubernetes não poderá satisfazer o pedido de dois `dongles`, porque o primeiro pod
usou três dos quatro `dongles` disponíveis.
This was after the LGTM label.

**Comment by stormqueen1990** [comment]
Hi there, @DonatoHorn! 👋

This PR is approved, but it cannot be merged if the build does not pass. Right now it is not passing because this branch is not up-to-date with the `main` branch from upstream. If you need help doing this rebase, please don't hesitate to reach out on Slack. 😃

**Comment by stormqueen1990** [comment]
Hi there, @DonatoHorn 👋 

It seems we hit the `build exceeded maximum allowed runtime` error in Netlify for this PR. Could you please try pushing an empty commit for triggering a new build with `git commit --allow-empty`?

As always, if you need any help please don't hesitate to reach out on Slack. 😃

**Comment by sftim** [comment]
I triggered a Netlify rebuild for you @DonatoHorn

**Comment by stormqueen1990** [comment]
> I triggered a Netlify rebuild for you @DonatoHorn

Hi there, @sftim!

Will this also update the PR checks?

This PR has been approved for a few weeks but was failing due to the CVE feed shortcode change, and due to that failure it never made it to the merge queue.

**Pull request merged**
